data_rentgen/server/schemas/v1/lineage.py

```diff
@@ -96,21 +77,21 @@ def _check_until(cls, value: datetime | None, info: ValidationInfo) -> datetime
             raise ValueError("'since' should be less than 'until'")
         return value
 
-    @model_validator(mode="after")
-    def _check_ids(self):
-        from uuid import UUID as OLD_UUID
-
-        if self.point_kind in {LineageEntityKindV1.JOB, LineageEntityKindV1.DATASET} and not isinstance(
-            self.point_id,
-            int,
-        ):
-            raise ValueError(f"'point_id' should be int for '{self.point_kind}' kind")
-        elif self.point_kind in {LineageEntityKindV1.RUN, LineageEntityKindV1.OPERATION} and not isinstance(
-            self.point_id,
-            OLD_UUID,
-        ):
-            raise ValueError(f"'point_id' should be UUIDv7 for '{self.point_kind}' kind")
-        return self
+
+class DatasetLineageQueryV1(BaseLineageQueryV1):
+    point_id: int = Field(description="Dataset id", examples=[42])
```

Comment:
What do you think about renaming `point_id` -> `start_at_id`?

Reply:
Mb `started_point_id` or `started_node_id`

Or you prefer without `point`?

Reply:
Let's use `start_node_id`